packages/cli/index.js

```diff
@@ -298,9 +298,7 @@ program
     const rootFolder = path.resolve(process.cwd());
     const outputRoot = path.join(rootFolder, '_site');
     new Site(rootFolder, outputRoot, undefined, undefined, options.siteConfig).deploy(options.travis)
-      .then(() => {
-        logger.info('Deployed!');
-      })
+      .then(deployMsg => logger.info(deployMsg))
```

Comment:
It doesn't feel very intuitive for the `Site` constructor to return a message. The message feels more like the responsibility of the CLI. I think that it might be more appropriate for it to return some data about the site created (the url in this case), which we can then use to construct the deploy message here.